observability/distr_tracing/distr_tracing_rn/distr-tracing-rn-3-2.adoc

```diff
@@ -0,0 +1,129 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="distributed-tracing-rn-3-2"]
+= Release notes for {DTProductName} 3.2
+:context: distributed-tracing-rn-3-2
+
+toc::[]
+
+include::modules/distr-tracing-product-overview.adoc[leveloffset=+1]
+
+You can use the {DTShortName} xref:../../otel/otel-forwarding.adoc#otel-forwarding-traces[in combination with] the xref:../../otel/otel-installing.adoc#install-otel[{OTELName}].
+
+This release of the {DTProductName} includes the {TempoName} and the deprecated {JaegerName}.
+
+////
+[id="distributed-tracing-rn_3-2_cves"]
+== CVEs
+
+This release fixes link:https://access.redhat.com/security/cve/cve-202?-?????[CVE-202?-?????].
+////
+
+[id="distributed-tracing-rn_3-2_tempo-release-notes"]
+== {TempoName}
+
+The {TempoName} is provided through the {TempoOperator}.
+
+[id="distributed-tracing-rn_3-2_tempo-release-notes_technology-preview-features"]
+=== Technology Preview features
+
+This update introduces the following Technology Preview feature:
+
+* Support for the Tempo monolithic deployment.
+
+:FeatureName: The Tempo monolithic deployment
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+[id="distributed-tracing-rn_3-2_tempo-release-notes_new-features-and-enhancements"]
+=== New features and enhancements
+
+This update introduces the following enhancements:
+
+* {TempoName} 3.2 is based on the open source link:https://grafana.com/oss/tempo/[Grafana Tempo] 2.4.1.
+* Allowing the overriding of resources per component.
+
+////
+[id="distributed-tracing-rn_3-2_tempo-release-notes_deprecated-functionality"]
+=== Deprecated functionality
+
+In the {TempoName} 3.2, ???
+////
+
+////
+[id="distributed-tracing-rn_3-2_tempo-release-notes_removal-notice"]
+=== Removal notice
+
+In {TempoName} 3.2, the FEATURE has been removed. Bug fixes and support are provided only through the end of the 3.? lifecycle. As an alternative to the FEATURE for USE CASE, you can use the ALTERNATIVE instead.
+////
+
+[id="distributed-tracing-rn_3-2_tempo-release-notes_bug-fixes"]
+=== Bug fixes
+
+This update introduces the following bug fixes:
```

Comment:
We need to add this bug fix for Tempo. https://issues.redhat.com/browse/TRACING-4087

Reply:
This does not seem to be reported by customers. Therefore I would skip it.